packages/dx-react-grid/src/plugins/table-filter-row.jsx

```diff
@@ -56,17 +56,21 @@ export class TableFilterRow extends React.PureComponent {
                       onValueChange: newValue => onFilter(newValue ? { value: newValue } : null),
                     }}
                   >
-                    {content => (
-                      <FilterCell
-                        {...params}
-                        getMessage={getMessage}
-                        column={params.tableColumn.column}
-                        filter={filter}
-                        onFilter={onFilter}
-                      >
-                        {content}
-                      </FilterCell>
-                    )}
+                    {(content) => {
+                      const filteringEnabled = isColumnFilteringEnabled(columnName);
+                      return (
+                        <FilterCell
+                          {...params}
+                          getMessage={getMessage}
+                          column={params.tableColumn.column}
+                          filter={filter}
+                          filteringEnabled={filteringEnabled}
```

Comment:
I'd prefer `disabled` here

Reply:
Well, we've already discussed the similar point [here](https://github.com/DevExpress/devextreme-reactive/pull/702#discussion_r164691061). And I'm not sure that `disabled` sounds good for a cell.